src/uiTest/resources/project.rhino.js

```diff
@@ -379,10 +393,75 @@ function Project(underlyingProject) {
     return component;
   };
 
+  this.toggleListingCommits = function () {
+    invokeActionAndWait('GitMachete.ToggleListingCommitsAction', ACTION_PLACE_TOOLBAR, {});
+  };
+
   this.checkoutBranch = function (branchName) {
     invokeActionAndWait('GitMachete.CheckoutSelectedAction', ACTION_PLACE_CONTEXT_MENU, { SELECTED_BRANCH_NAME: branchName });
   };
 
+  this.fastForwardMergeSelectedToParent = function (branchName) {
```

Comment:
Okay... so the existing `toolbar`/`contextMenu` will remain in Rhino codebase? 🤔 I'm afraid they'll _rot_ quite soon, being even less reliable over time (with new changes to the UI etc.) then they're now :/

Reply:
I'd rather remove them completely TBH